docs/Systems.md

```diff
@@ -387,12 +387,10 @@ Because Buildroot can only be used from Linux, Nerves provides an abstraction
 layer called the Nerves system configuration shell that allows the same
 procedure to be used on Linux and non-Linux development hosts by using a
 Linux-based Docker container on non-Linux platforms. To access this environment,
-run the `mix nerves.system.shell` task, either from your project directory or
-from the custom system source directory.
+run the `mix nerves.system.shell` task from the custom system source directory.
 
 ```bash
-$ export MIX_TARGET=rpi3
-$ mix deps.get
+$ MIX_TARGET=rpi3 mix deps.get
```

Comment:
Should this also be `custom_rpi3` instead of just `rpi3` as well?

Reply:
Yes sorry, I should have been more clear about that. I think the answer is yes but I haven’t had a chance to try it out myself to verify, so it’s also in the “note to self” category. :smile_sweat: